src/style.css

```diff
@@ -1,3 +1,179 @@
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --white: #fff;
+  --lightgrey: #f3f3f3;
+  --darkgrey: #060b35;
+  --grey: #616070;
+  --blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
It's a bad practice to hardcode height. The height should be calculated by the browser depending on the content height